docs/application/dotnet/guides/app-management/component-port.md

```diff
@@ -17,7 +17,7 @@ The main feature of the `Tizen.Applications.ComponentBased.ComponentPort` class
 To enable your component to use the component port functionality, follow these steps:
 
 1.  You need two components to communicate with each other through the component port.
-2.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentPort](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html){:target="_blank"} and [Tizen.Applications.ComponentBased.ComponentTask](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.ComponentPort.html){:target="_blank"} classes, include the [Tizen.Applications.ComponentBased](https://samsung.github.io/TizenFX/API9/api/Tizen.Applications.ComponentBased.html){:target="_blank"} namespace in your component:
+2.  To use the methods and properties of the [Tizen.Applications.ComponentBased.ComponentPort](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.ComponentPort.html) and [Tizen.Applications.ComponentBased.ComponentTask](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.ComponentPort.html) classes, include the [Tizen.Applications.ComponentBased](/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.html) namespace in your component:
```

Comment:
Thanks for the update!

However I just found one more suspicious point here.
I think when you see added hyperlinks, it would be better to check always if the URL is valid.
I just wish if this link had been checked with the author when the author added this line.

In this line, there are 3 hyperlinks and I doubt one is going to incorrect page.

- \[Tizen.Applications.ComponentBased.**ComponentPort**]\(/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.**ComponentPort**.html)

    It is talking about ComponentPort, and is going to ComponentPort page.

- \[Tizen.Applications.ComponentBased.**ComponentTask**]\(/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.**ComponentPort**.html)

    It is talking about ComponentTask, and is going to ComponentPort page...?
    Also you can find [ComponentTask](https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.ComponentTask.html) near [the API page guided](https://docs1.stg.tizen.org/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.ComponentBased.ComponentPort.html
). 
- \[Tizen.Applications.**ComponentBased**]\(/application/dotnet/api/TizenFX/latest/api/Tizen.Applications.**ComponentBased**.html)

    It is talking about ComponentBased, and is going to ComponentBased page.

Reply:
Thanks again for another update @omar-srbd 
All my comments are resolved.